pkgs/servers/web-apps/healthchecks/default.nix

```
@@ -0,0 +1,86 @@
+{ lib
+, writeText
+, fetchFromGitHub
+, nixosTests
+, python3
+}:
+let
+  py = python3.override {
+    packageOverrides = self: super: {
```

```suggestion
    packageOverrides = final: prev: {
```
newer things use those better named variables.